### DISCUSSION TITLE
Velero can't sync more than about the thousand backup CRDs from object storage

### DISCUSSION THREAD

**Discussion by averti** [vmware-tanzu/velero#3643] [Community support Q&A]
Velero can't sync more than about the thousand backup CRDs from object storage. 
I see all my backups in object storage and they created without errors, but after sync with s3 storage new crd backups.velero.io disappered. After removing old backups, new crd backups.velero.io appear.
How can I increase this limit? 

Client:
        Version: v1.5.3
        Git commit: 123109a3bcac11dbb6783d2758207bac0d0817cb
Server:
        Version: v1.5.3
        
For example:
root@ubuntu:/# velero get backups | wc -l
1000
root@ubuntu:/# velero backup create test-full-backup --include-namespaces test
Backup request "test-full-backup" submitted successfully.
Run `velero backup describe test-full-backup` or `velero backup logs test-full-backup` for more details.
root@ubuntu:/# velero get backups | wc -l
1001
root@ubuntu:/# velero get backups | grep test-full-backup | wc -l
1
root@ubuntu:/# kubectl -n velero logs velero-5f646f95cd-d8vv5 | grep test-full-backup | grep Deleted
time="2021-03-30T14:48:01Z" level=debug msg="Deleted orphaned backup from cluster" backup=test-full-backup backupLocation=default controller=backup-sync logSource="pkg/controller/backup_sync_controller.go:342"

root@ubuntu:/# velero get backups | grep test-full-backup | wc -l
0
root@ubuntu:/# velero get backups | wc -l
1000
root@ubuntu:/# velero delete backup old-test-full-backup --confirm
Request to delete backup "old-test-full-backup" submitted successfully.
The backup will be fully deleted after all associated data (disk snapshots, backup files, restores) are removed.
root@ubuntu:/# velero get backups | grep test-full-backup | wc -l
1

**Comment by jenting** [comment]
Which plugin you're used? aws/gcp/azure?

**Reply by averti** [reply]
aws
